openest/lincombo/hierregress.py

```diff
@@ -31,7 +31,7 @@ def probability_tau(mus, tau, yy, stdvars, XX, probability_prior_tau):
     mv_betahat_given_tau = betahat_given_tau(yy, stdvars, XX, tau)
     mv_mu_given_tau = mu_given_tau(yy, stdvars, XX, tau)
 
-    betahats = np.dot(XX, np.transpose(np.matrix(mus)))
+    betahats = np.dot(XX, np.transpose(np.atleast_2d(mus)))
```

Comment:
How could `mus` have more than 2 dims?

Reply:
By using `np.matrix()` in the original code, you're explicitly making it a 2D "array". I needed to ditch `np.matrix` because it's deprecated. Replacing it with `np.atleast_2d`, I feel, keeps with the original spirit of the `np.matrix` and lets `mus` get transposed.

Reply:
Okay, that's fine. Didn't see before that matrix was being deprecated.